### DISCUSSION TITLE
custom font breaks urls

### DISCUSSION THREAD

**Discussion by noyloza** [bootscore/bootscore#231] [General]
This might be a very easy or simple issue, but for some reason when I use Bootscore and use a custom font, all my urls except the main nav become static and unclickable. Anybody have that issue before?

**Comment by crftwrk** [comment]
This sounds weird, never heard this before. But for sure that this has nothing to do with the theme, it must be something wrong with the font. However, I can't reproduce this quickly, we need more information in detail:

- Share a link to your project to check this bug. If you work on your local computer, transfer project to a public server.
- Describe exactly which font(s) you use and share the download links to them.
- Share the code snippet how you added font(s) to your theme.

If you are using Google Fonts, check here how to install them in bootScore https://bootscore.me/documentation/google-fonts/

**Reply by crftwrk** [reply]
I see. Gotham font is broken, Avenir Next LT Pro works well. Tested it with a simple HTML document as well, it's not the theme.

However, both fonts aren't free (not cheap) and the cdn links are for sure not official. Why not install similar Google fonts legal and safe on your site?

- Gotham https://similarfont.io/3-google-fonts-similar-to-gotham
- Avenir https://similarfont.io/3-google-fonts-similar-to-avenir-next

**Comment by noyloza** [comment]
Thank you for that help, I will change fonts!
